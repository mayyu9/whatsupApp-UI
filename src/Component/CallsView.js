import React from 'react';
import '../App.css';

const CallsView = (props) => {
  const finalCalls = props.calls;
  return(
    <div>
    {finalCalls ? finalCalls.map((data,index) => {return<ul key={index+1}><li>{data}</li></ul>}): 'calls'}
    </div>
  );
};

export default CallsView;
