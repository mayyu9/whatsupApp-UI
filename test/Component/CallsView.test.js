import React from 'react';
import { shallow, mount } from 'enzyme';
import CallsView from '../../src/Component/CallsView';
import mock from '../../src/MockData/Mockdata';

it('should render calls view', () => {
  const view = shallow(<CallsView calls={mock.Calls}/>);
  expect(view).toMatchSnapshot();
});
