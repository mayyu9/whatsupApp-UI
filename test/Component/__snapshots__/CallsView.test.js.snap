// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render calls view 1`] = `
<div>
  <ul
    key="1"
  >
    <li>
      mom
    </li>
  </ul>
  <ul
    key="2"
  >
    <li>
      grandMaa
    </li>
  </ul>
  <ul
    key="3"
  >
    <li>
      papa
    </li>
  </ul>
  <ul
    key="4"
  >
    <li>
      mama
    </li>
  </ul>
  <ul
    key="5"
  >
    <li>
      mami
    </li>
  </ul>
  <ul
    key="6"
  >
    <li>
      pavan
    </li>
  </ul>
  <ul
    key="7"
  >
    <li>
      sagar
    </li>
  </ul>
  <ul
    key="8"
  >
    <li>
      yashu
    </li>
  </ul>
  <ul
    key="9"
  >
    <li>
      murali
    </li>
  </ul>
  <ul
    key="10"
  >
    <li>
      keerthi
    </li>
  </ul>
</div>
`;
